common/BUILD

```diff
@@ -22,14 +22,15 @@ package(default_visibility = ["//visibility:public"])
 
 java_library(
     name = "common",
-    srcs = glob(["*/*.java"], exclude=["test/*", "*Test.java", "*/*Test.java"]),
+    srcs = glob(["*/*.java", "*/*/*.java"], exclude=["test/*", "*Test.java", "*/*Test.java"]),
     deps = [
         # Internal Repository Dependencies
         "@graknlabs_common//:common",
 
         # External Maven Dependencies
         "@maven//:com_github_ben_manes_caffeine_caffeine",
         "@maven//:com_google_code_findbugs_jsr305",
+        "@maven//:org_slf4j_slf4j_api",
```

Comment:
Adding a logger needed by the actor (the actor is designed to log unhandled error)